### ISSUE TITLE
Node.js Foundation Technical Steering Committee (TSC) Meeting 2018-10-31

### ISSUE THREAD

**Issue by mhdawson** [nodejs/TSC#618]
## Time

**UTC Wed 31-Oct-2018 16:00 (04:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Wed 31-Oct-2018 09:00 (09:00 AM) |
| US / Mountain | Wed 31-Oct-2018 10:00 (10:00 AM) |
| US / Central  | Wed 31-Oct-2018 11:00 (11:00 AM) |
| US / Eastern  | Wed 31-Oct-2018 12:00 (12:00 PM) |
| London        | Wed 31-Oct-2018 16:00 (04:00 PM) |
| Amsterdam     | Wed 31-Oct-2018 17:00 (05:00 PM) |
| Moscow        | Wed 31-Oct-2018 19:00 (07:00 PM) |
| Chennai       | Wed 31-Oct-2018 21:30 (09:30 PM) |
| Hangzhou      | Thu 01-Nov-2018 00:00 (12:00 AM) |
| Tokyo         | Thu 01-Nov-2018 01:00 (01:00 AM) |
| Sydney        | Thu 01-Nov-2018 03:00 (03:00 AM) |

Or in your local time:
* http://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Technical%20Steering%20Committee%20(TSC)+Meeting+2018-10-31&iso=20181031T16
* or http://www.wolframalpha.com/input/?i=04PM+UTC%2C+Oct+31%2C+2018+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/15C1wCm2IoIUgbl49KqQK88hllyZ0F86Ej242pmCRRXA/edit>

## Agenda

Extracted from **tsc-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

### nodejs/node

* fs: behaviour of readFile and writeFile with file descriptors [#23433](https://github.com/nodejs/node/issues/23433)
  * Vote has been suggested.

### nodejs/TSC

* Tracking issue for updating TSC on Board Meetings [#476](https://github.com/nodejs/TSC/issues/476)
* Strategic Initiatives - Tracking Issue [#423](https://github.com/nodejs/TSC/issues/423)

## Invited

* Anna Henningsen @addaleax (TSC)
* Anatoli Papirovski @apapirovski (TSC)
* Сковорода Никита Андреевич @ChALkeR (TSC)
* Colin Ihrig @cjihrig (TSC)
* Daniel Bevenius @danbev (TSC)
* Franziska Hinkelmann @fhinkel (TSC)
* Gabriel Schulhof @gabrielschulhof (TSC)
* Gibson Fahnestock @gibfahn (TSC)
* Jeremiah Senkpiel @Fishrock123 (TSC)
* James M Snell @jasnell (TSC)
* Joyee Cheung @joyeecheung (TSC)
* Matteo Collina @mcollina (TSC)
* Michael Dawson @mhdawson (TSC)
* Myles Borins @MylesBorins (TSC)
* Ali Ijaz Sheikh @ofrobots (TSC)
* Rod Vagg @rvagg (TSC)
* Michaël Zasso @targos (TSC)
* Sakthipriyan Vairamani @thefourtheye (TSC)
* Timothy Gu @TimothyGu (TSC)
* Rich Trott @Trott (TSC)

## Notes

The agenda comes from issues labelled with `tsc-agenda` across **all of the repositories in the nodejs org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

Zoom link: https://zoom.us/j/611357642
Regular password

## Public participation

We stream our conference call straight to YouTube so anyone can listen to it live, it should start playing at **https://www.youtube.com/c/nodejs+foundation/live** when we turn it on. There's usually a short cat-herding time at the start of the meeting and then occasionally we have some quick private business to attend to before we can start recording & streaming. So be patient and it should show up.

Many of us will be on IRC in #node-dev on Freenode if you'd like to interact, we have a Q/A session scheduled at the end of the meeting if you'd like us to discuss anything in particular. @nodejs/collaborators in particular if there's anything you need from the TSC that's not worth putting on as a separate agenda item, this is a good place for that.

---

**Invitees**

Please use the following emoji reactions in this post to indicate your
availability.

* :+1: - Attending
* :-1: - Not attending
* :confused: - Not sure yet

**Comment by addaleax** [comment]
I’m unlikely to make it, but I think our onboarding rules want some kind of announcement for collaborator nominations – currently open are https://github.com/nodejs/node/issues/23849, https://github.com/nodejs/node/issues/23850 and https://github.com/nodejs/node/issues/23955… maybe somebody could make sure that these are mentioned? :)

**Comment by thefourtheye** [comment]
https://github.com/nodejs/node/issues/23433 is not added to the agenda. I propose moving that to a vote.

**Comment by mhdawson** [comment]
I won't be able to make the meeting next week as I'll be at NodeConfEU.  Any volunteers to chair?  It is at 4EST.

**Comment by cfanoulis** [comment]
I'd like to know whether Athens time (GMT +2, 18:00) can be added to the time chart.

**Comment by Trott** [comment]
It's going to be hard to vote on nodejs/node#23433 because there are 5 options. 

There is no reason to vote in the meeting, and in fact, we should avoid meeting votes in my opinion. (Our governance doc even says so, or used to. Not seeing it now.) We should initiate the vote in the issue tracker. I'm not sure how you want to handle it with 5 options, though. I suggest someone (preferably @thefourtheye) champion a single option (maybe based on feedback in the issue) and we have an up-or-down vote on that one single option.

**Comment by richardlau** [comment]
> 
> 
> I'd like to know whether Athens time (GMT +2, 18:00) can be added to the time chart.

You could raise a pull request/issue against https://github.com/nodejs/make-node-meeting:
https://github.com/nodejs/make-node-meeting/blob/756ecd246f87eccad97d3681adbb42043d68ddbd/make-node-meeting.sh#L64-L76

**Comment by cfanoulis** [comment]
:+1: Will do, its October so another PR cannot hurt

**Comment by mhdawson** [comment]
@cfanoulis to add another time you wound need to PR a change into node-meeting-agenda or make-node-meeting which are modules used by https://github.com/nodejs/create-node-meeting-artifacts to create the items for the meetings.

**Comment by Trott** [comment]
Do note that there are already two links below the chart for getting the meeting time in your local time zone.

I won't stop someone from adding a time zone but if it were left up to me, I'd prefer we *remove* time zones rather than add them. A comprehensive list of times across the globe isn't going to happen. In the past, it listed just time zones that had TSC members living in them. The table is long and a bit unwieldy, and there are those links I just mentioned. Based on the locations of TSC members, I'm guessing we could do fine without Mountain, Central, London, and Tokyo.

**Comment by richardlau** [comment]
> 
> 
> Do note that there are already two links below the chart for getting the meeting time in your local time zone.
> 
> I won't stop someone from adding a time zone but if it were left up to me, I'd prefer we _remove_ time zones rather than add them. A comprehensive list of times across the globe isn't going to happen. In the past, it listed just time zones that had TSC members living in them. The table is long and a bit unwieldy, and there are those links I just mentioned. Based on the locations of TSC members, I'm guessing we could do fine without Mountain, Central, London, and Tokyo.

I believe the tool is not TSC specific and uses the same list of time zones for any meeting it creates. At least the London timezone was added for CommComm (https://github.com/nodejs/make-node-meeting/issues/20).

**Comment by Trott** [comment]
> I believe the tool is not TSC specific and uses the same list of time zones for any meeting it creates. 

Good point and something that had escaped my notice...

**Comment by thefourtheye** [comment]
@nodejs/tsc There is a table in https://github.com/nodejs/node/issues/23433, which we can use to vote for it. Abstainers can just add an empty row with their name. As the options are not binary, this could be used as an alternative to the formal voting.

**Comment by cfanoulis** [comment]
@Trott replying to your comment:

By Athens, I mean the GMT +2 (UTC +3) timezone, which seems to be absent, or actually skipped. I will agree with your comment that there may be a few too many timezones, but since GMT +1/3/4 exist, I would say that it could not be that bad to add a timezone.

**Comment by targos** [comment]
I'll actually have to miss this meeting because of the DST shift :(
I updated my slots in the sheet.

**Comment by mhdawson** [comment]
Once the TZ's settle I was planning on asking everybody to update the sheet so we can take another look.,=

**Comment by ChALkeR** [comment]
@mhdawson, I'm be planning on adding TZ support :-). A bit later, though.

**Comment by mhdawson** [comment]
PR for minutes: https://github.com/nodejs/TSC/pull/619

**Comment by thefourtheye** [comment]
Moderation Team Updates:

* An odd PR was reported. It was closed and marked as invalid to prevent Hacktoberfest Participation (https://github.com/nodejs/moderation/issues/240)
* Two spammy comments were removed (https://github.com/nodejs/moderation/issues/242)
* @Trott Stepped off the Moderation Team this week.

@nodejs/tsc @nodejs/community-committee @nodejs/moderation
